src/models/legend.js

```diff
@@ -38,43 +38,38 @@ nv.models.legend = function() {
                 })
                 .on('mouseout', function(d,i) {
                     dispatch.legendMouseout(d,i);
-                })
-                .on('click', function(d,i) {
-                    dispatch.legendClick(d,i);
-                    if (updateState) {
-                        if (radioButtonMode) {
-                            //Radio button mode: set every series to disabled,
-                            //  and enable the clicked series.
-                            data.forEach(function(series) { series.disabled = true});
-                            d.disabled = false;
-                        }
-                        else {
-                            d.disabled = !d.disabled;
-                            if (data.every(function(series) { return series.disabled})) {
-                                //the default behavior of NVD3 legends is, if every single series
-                                // is disabled, turn all series' back on.
-                                data.forEach(function(series) { series.disabled = false});
-                            }
+                });
+            if(isTouchable) {
```

Comment:
if PC device, use click/dblclick.
if touchable device, use touchstart/taphold.
Taphold is a long touch more than 1 second.